### PR TITLE
Document clusterwide configuration

### LINKED ISSUE #569: Describe the clusterwide config concept
What clusterwide config is? How is it represented? How is it stored? How is it applied? Mention 2pc. How can it be modified (http/graphql/luatest)? Provide API usage examples. 

https://www.tarantool.io/en/doc/2.2/book/cartridge/cartridge_admin/#updating-the-configuration

### LINKED ISSUE #716: Document config upload
How do I upload configuration with curl? Provide an example

### PR DESCRIPTION
**Pull request by HustonMmmavr**
Close #716
Close #569

### CODE REVIEW

File: topics/clusterwide-config.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+ports, workdirs, memory settings etc.
+
+## Internal representation
+
+On file system clusterwide configuration is represented by a **file
+tree**. Inside `workdir` of any configured instance one can find the
+following directory:
+
+```text
+config/
```
Comment (olegrok) [inline topics/clusterwide-config.md:22]: I think we should add `.tarantool.cookie` here.

**Comment by olegrok** [comment]
This feature is not describe - https://github.com/tarantool/cartridge/commit/ef9948e8e96e6c0687077a2ca7d967a4f8dd29ef

File: topics/clusterwide-config.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+some other **role-specific data**. It supports YAML as well as plain
+text sections. And it can also be organized in nested subdirectories.
+
+Role-specific sections are used by some third-party roles, i.e.
+[sharded-queue](https://github.com/tarantool/sharded-queue) and
+[cartridge-extensions](https://github.com/tarantool/cartridge-extensions).
+
+A user can influence clusterwide configuration in various ways. One can
+alter configuration using Lua, HTTP or GraphQL API. Also there are
+Luatest helpers available.
```
Comment (olegrok) [inline topics/clusterwide-config.md:84]: Shouldn't we refer luatest module here?

File: topics/clusterwide-config.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+---
+custom_section: {}
+...
+CONFIG
+```
+
+Upload new config:
+
+```bash
+curl -v "localhost:8081/admin/config" -X PUT --data-binary @config.yml
```
Comment (olegrok) [inline topics/clusterwide-config.md:104]: Authorization is disabled and I think it should be mentioned

File: topics/clusterwide-config.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+Download it:
+
+```bash
+curl -v "localhost:8081/admin/config" -o config.yml
+```
+
+It's suitable for role-specific sections only. System sections
+(`topology`, `auth`, `vshard_groups`, `users_acl`) can't be neither
+uploaded nor downloaded.
```
Comment (olegrok) [inline topics/clusterwide-config.md:115]: and modified

Comment (olegrok) [reply topics/clusterwide-config.md:115]: but it's relevant for cartridge.config_patch_clusterwide...

File: topics/clusterwide-config.md (hunk 1 above)
Comment (rosik) [reply topics/clusterwide-config.md:22]: It's not a part of clusterwide configuration.

File: topics/clusterwide-config.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+some other **role-specific data**. It supports YAML as well as plain
+text sections. And it can also be organized in nested subdirectories.
+
+Role-specific sections are used by some third-party roles, i.e.
+[sharded-queue](https://github.com/tarantool/sharded-queue) and
+[cartridge-extensions](https://github.com/tarantool/cartridge-extensions).
+
+A user can influence clusterwide configuration in various ways. You can
+alter configuration using Lua, HTTP or GraphQL API. Also there are
+(luatest)[https://github.com/tarantool/luatest] helpers available.
```
Comment (olegrok) [inline topics/clusterwide-config.md:85]: It looks as 
image

**Pull request merged**